GitUI/Avatars/InitialsAvatarProvider.cs

```diff
@@ -65,22 +79,42 @@ private static (string? name, char[]? separator) NameSelector(string? name, stri
                 return null;
             }
 
+            string name = names[0];
+
             // If only a single valid name-element is found ...
             if (names.Length == 1)
             {
                 // ... and that name-element is only a single character long ...
-                if (names[0].Length == 1)
+                if (name.Length == 1)
                 {
                     // ... return that character as uppercase
-                    return names[0][0].ToString().ToUpper();
+                    return name[0].ToString().ToUpper();
+                }
+
+                if (char.IsUpper(name[1]))
+                {
+                    return $"{char.ToUpper(name[0])}{name[1]}";
+                }
+
+                var splitNames = name.Split(_emailInitialSeparator);
```

Comment:
[nit] avoid `var` unless the type is obvious

Reply:
done.